internal/command/flags.go

```diff
@@ -36,4 +36,9 @@ var Flags = []cli.Flag{
 		Name:  "ignore-all, I",
 		Usage: "Ignore inaccessible or missing keys",
 	},
+
+	cli.BoolFlag{			//<----------------------------
+		Name:  "version-providers, V",
```

Comment:
This flag name is a bit confusing since it almost implies that you are wanting to set a version on the providers. Maybe `--provider-versions` or `--all-versions` instead? I don't have a good idea of what we should use but the current name does need to change.

Reply:
`--all-provider-versions` and `-V` ?

Reply:
works for me (though maybe the `all` is a touch superfluous if they specify a provider but up to you)